### PR TITLE
UI

### PR DESCRIPTION
**Pull request by ClaudiuChelcea**
Add main menu. Not complete, but adding it so there wont be conflicts.

### CODE REVIEW

**Comment by RazvanRotaru** [comment]
539 changed files? :(

**Comment by ClaudiuChelcea** [comment]
There are many assets, packages, scripts. I didn`t change the current ones, just added a new scene that required a lot of things.

**Comment by ClaudiuChelcea** [comment]
So is it a yes or a no? @RazvanRotaru

File: Assets/Resources/BOXOPHOBIC/Skybox Cubemap Extended/Core/Editor/SkyboxExtendedWindows.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+public static class SkyboxExtendedWindows
```
Comment (RazvanRotaru) [inline Assets/Resources/BOXOPHOBIC/Skybox Cubemap Extended/Core/Editor/SkyboxExtendedWindows.cs:4]: what is this class doing?

**Review by RazvanRotaru — APPROVED** [review]
I believe there are resources we won't need but we'll ditch them later.

**Comment by ClaudiuChelcea** [comment]
Removed unused resources.

File: Assets/Resources/BOXOPHOBIC/Skybox Cubemap Extended/Core/Editor/SkyboxExtendedWindows.cs (hunk 1 above)
Comment (ClaudiuChelcea) [reply Assets/Resources/BOXOPHOBIC/Skybox Cubemap Extended/Core/Editor/SkyboxExtendedWindows.cs]: Probably skybox, it`s imported.

**Pull request merged**